next.config.js

```diff
@@ -0,0 +1,24 @@
+const { webpackDirAlias } = require('./dirAlias');
+const MomentTimezoneInclude = require('./src/app/legacy/psammead/moment-timezone-include/src');
+
+module.exports = {
+  reactStrictMode: true,
+  compiler: {
+    emotion: true,
+  },
+  eslint: {
+    ignoreDuringBuilds: true,
+  },
+  webpack: config => {
+    config.resolve.alias = {
+      ...webpackDirAlias,
+      ...config.resolve.alias,
+    };
+
+    config.plugins.push(
+      new MomentTimezoneInclude({ startYear: 2010, endYear: 2025 }),
+    );
+
+    return config;
+  },
+};
```

Comment:
I know we wanted to do as little custom configuration here as possible, but there's a few things going on here that are unavoidable without massive changes to the project.

1. Simorgh uses Emotion for styling, so that needs to be enabled.
2. Simorgh uses custom import aliases (ie. `import ArticlePage from '#pages/ArticlePage/ArticlePage'`) to avoid dot pathing everything, so the Webpack config is enhanced to accommodate this.
3. `MomentTimezoneInclude` is a custom Webpack plugin written by the SImorgh team to limit the size of timezones included to reduce bundle size. 
4. And #3 is why we need to disable lint during builds. It's a cyclical problem. We need Webpack to build the app, so the custom `MomentTimezoneInclude` runs to create the `tz` directory with the outputted timezones that are imported elsewhere, but we cannot build the app until it passes lint because imports fail to find `tz` files.